docs/scos/dev/front-end-development/migration-guide-switch-to-npm.md

```diff
@@ -0,0 +1,156 @@
+```
+
+### 2) Update configs
+
+1. Set up a new versions of node/npm in the main `*.yml` files, like `deploy.yml`, `deploy.dev.yml` and `deploy.ci.yml`:
+
+```yaml
+image:
+    ...
+    node:
+        version: 16
+        npm: 8
+```
+
+{% info_block infoBox "Note" %}
+To make sure the CI jobs will pass deploy, add same part of config to all 'deploy.*.yml' files, where frontend are using.
+{% endinfo_block %}
+
+2. Update docker hash in the `.git.docker` file to make sure the correct version of `docker-sdk` will be installed:
+
+```text
+e9ebb666feccae1754792d41e49df3b9f95ef0aa
+```
+
+Then run next command to pull latest `docker-sdk` version:
+
+```bash
+cd docker && git pull origin master
+```
+
+3. Update `package.json`:
+
+```json
+{
+    ...
+    "engines": {
+        "node": ">=16.0.0",
+        "npm": ">=8.0.0"
+    },
+    "dependencies": {
+        ...
+        "@babel/plugin-proposal-class-properties": "~7.17.12",
+        "@babel/plugin-transform-runtime": "~7.17.12",
+        "@babel/preset-typescript": "~7.17.12",
+        "@spryker/oryx-for-zed": "~2.12.0",
+        "babel-loader": "~8.2.5",
+        "sass": "~1.51.0",
+        ...
+    },
+    ...
+}
+```
+
+4. Update `.travis.yml`:
+
+```yaml
+before_install:
+  ...
+  - nvm use 16
+  ...
+```
+
+5. Create a new `.npmrc` file in the root directory with the next content: 
+
+```text
+legacy-peer-deps=true
+```
+
+6. Use single command to install dependencies `frontend:project:install-dependencies` in `*.yml` files instead of `frontend:{yves/zed/mp}:install-dependencies`.
+
+7. Delete the following folders/files from the root directory:
+
+- `.yarn` folder
+- `.yarnrc.yml` file
+- `yarn.lock` file
+
+### 3) Build the project
+
+1. Run the following commands to apply the docker changes:
+
+```bash
+docker/sdk boot deploy.dev.yml
+docker/sdk up
+```
+
+2. Regenerate `package-lock.json`: 
+
+```bash
+rm -rf package-lock.json
+docker/sdk cli npm install
+```
+
+3. Run the following commands to build the project using NPM v8:
+
+```bash
+rm -rf node_modules && docker/sdk cli rm -rf node_modules
+docker/sdk boot deploy.dev.yml
+docker/sdk up --build --assets --data
+```
+
+{% info_block infoBox "Note" %}
+The following commands are deprecated and will work correctly only with the previous versions (using Yarn):
+
+```bash
+frontend:yves:install-dependencies
+frontend:zed:install-dependencies
+frontend:mp:install-dependencies
+```
+
+To pass an installation of all (yves, zed, mp) frontend dependencies use a single command: 
+
+```bash
+frontend:project:install-dependencies
+```
+{% endinfo_block %}
+
+{% info_block infoBox "Note" %}
+If 'node' and 'npm' are uses locally, make sure their versions are correct:
+
+```bash
+node -v
+npm -v
+```
+
+To update them use the following commands:
+
+```bash
+sudo npm -g install npm@8
+sudo npm cache clean -f
+sudo npm install -g n
+sudo n 16
+```
```

Comment:
I think we shouldn't provide commands for updating other tools.
Please add links to the appropriate docs.